internal/service/ecs/service.go

```diff
@@ -721,8 +721,10 @@ func resourceServiceRead(d *schema.ResourceData, meta interface{}) error {
 		log.Printf("[ERR] Error setting placement_constraints for (%s): %s", d.Id(), err)
 	}
 
-	if err := d.Set("network_configuration", flattenEcsNetworkConfiguration(service.NetworkConfiguration)); err != nil {
-		return fmt.Errorf("error setting network_configuration for (%s): %w", d.Id(), err)
+	if service.DeploymentController == nil || aws.StringValue(service.DeploymentController.Type) != ecs.DeploymentControllerTypeExternal {
```

Comment:
this is copy pasted in from current code at https://github.com/figma/terraform-provider-aws/blob/figma/aws/resource_aws_ecs_service.go#L722